configuration/configuration_whitebox_test.go

```diff
@@ -9,9 +10,35 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+var reqLong *goa.RequestData
+var reqShort *goa.RequestData
+var isConfigurationSet bool
+
+func setup() {
```

Comment:
I think we can use `init` here. So, no need to call `setup` explicitly everywhere.